druid/src/contexts.rs

```diff
@@ -103,6 +105,13 @@ pub struct LayoutCtx<'a, 'b: 'a> {
     pub(crate) window_id: WindowId,
 }
 
+/// Z-order paint operations with transformations.
+pub struct ZOrderPaintOp {
```

Comment:
Does this need to be public? It feels like an implementation detail. Maybe just `pub(crate)`?